src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -264,29 +274,26 @@ private static ServerMessage endCurrentPlayerTurn(WebSocketPlayer player) {
         return initiateRound(player);
     }
 
-    private static ServerMessage cheating(int totalValue, WebSocketPlayer player){
+    private static ServerMessage cheating(int totalValue, WebSocketPlayer player) {
         Game game = player.getCurrentGame();
         Dice dice = player.getCurrentGame().getDice();
 
-        dice.setValue1(totalValue/2);
-        dice.setValue2(totalValue - totalValue/2);
+        dice.setValue1(totalValue / 2);
+        dice.setValue2(totalValue - totalValue / 2);
 
         game.cheating();
 
-        return ServerMessage.builder()
-                .messagePath("cheating")
-                .type(ServerMessage.MessageType.INFO)
-                .game(game)
-                .build();
+        return ServerMessage.builder().messagePath("cheating").type(ServerMessage.MessageType.INFO).game(game).build();
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason